packages/adapters/wagmi/src/tests/client.test.ts

```diff
@@ -97,6 +101,38 @@ describe('WagmiAdapter', () => {
       expect(adapter.namespace).toBe('eip155')
     })
 
+    it('should set wagmi connectors', () => {
+      const addWagmiConnector = vi.spyOn(adapter, 'addWagmiConnector' as any)
```

Comment:
This is a nit and maybe hard thing, but I think we should not test private methods like that, we should test private methods through the public methods that call them, thats how it happens in the practice. https://softwareengineering.stackexchange.com/a/100966

Reply:
Yeah, maybe we should just initialize an appkit instance and see if it's calling this internally as it should

Reply:
Good idea, yeah i'll do that.